docker-compose/docker-compose-injistack/config/mimoto-default.properties

```diff
@@ -260,7 +260,7 @@ mosip.openid.issuers=mimoto-issuers-config.json
 mosip.openid.htmlTemplate=credential-template.html
 mosip.oidc.client.assertion.type=urn:ietf:params:oauth:client-assertion-type:jwt-bearer
 mosip.oidc.p12.filename=oidckeystore.p12
-mosip.oidc.p12.password=mosip123
+mosip.oidc.p12.password=xy4gh6swa2i
```

Comment:
Do we need to hard code the collab password ?  This is subjected to change as per security pocilies.